app/Platform/Actions/ResumePlayerSession.php

```diff
@@ -104,7 +107,7 @@ public function execute(
         $playerSession = new PlayerSession([
             'user_id' => $user->id,
             'game_id' => $game->id,
-            'game_hash_id' => $gameHash?->id,
+            'game_hash_id' => $gameHash && !$isForMultiDiscGamesSystem ? $gameHash->id : null,
```

Comment:
It's probably still useful to capture the first hash loaded.

Reply:
Adjusted in latest.